dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -179,6 +179,8 @@ log4j-1.2-api/2.21.0//log4j-1.2-api-2.21.0.jar
 log4j-api/2.21.0//log4j-api-2.21.0.jar
 log4j-core/2.21.0//log4j-core-2.21.0.jar
 log4j-slf4j2-impl/2.21.0//log4j-slf4j2-impl-2.21.0.jar
+logback-classic/1.2.10//logback-classic-1.2.10.jar
+logback-core/1.2.10//logback-core-1.2.10.jar
```

Comment:
Oh. Can we exclude this by reusing our existing log4j2 dependency?

Reply:
hmm.. zookeeper Migrate to Logback https://issues.apache.org/jira/browse/ZOOKEEPER-4427 in version 3.8.0 There are others that have asked the same question as you. https://github.com/apache/zookeeper/pull/1793#issuecomment-1557181105 but that will be a lot of work. Should we just go back to version 3.7.2 for this one?

Reply:
- Thank you for the confirmation, @xhumanoid .
- Thank you for updating the PR, @bjornjorgensen . Please revisit the PR title and description according to the code change.